ledger/complete/mtrie/forest_test.go

```diff
@@ -22,7 +22,7 @@ import (
 
```

Comment:
⚠️ 
Similar to my comment from above, testing the leaf construction and compactification and expansion of the trie is much more important than testing only with compactified leafs (which happens if we only use storage path of the form `xzy000...0`, as is now the case).

 

#### Suggestion:
change 
https://github.com/onflow/flow-go/blob/3b8b3e3b8e96028f0ff414bd6d92b19b109a5c6f/ledger/complete/mtrie/forest_test.go#L1028-L1032
to write the values into the bytes with the _largest_ indices (instead of the smallest)

Reply:
applied as suggested